OpenRA.Mods.Common/Traits/AutoCrusher.cs

```diff
@@ -0,0 +1,82 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2022 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Linq;
+using OpenRA.Primitives;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	class AutoCrusherInfo : PausableConditionalTraitInfo
+	{
+		[Desc("Maximum scan range for AutoCrusher.")]
```

Comment:
```suggestion
		[Desc("Maximum scan range to scan for targets.")]
```

Don't repeat the trait name in the description.

Reply:
should be all fixed